go-runtime/compile/external-module-template/_ftl/go/modules/{{ range .NonMainModules }}{{ push .Name . }}{{ end }}/external_module.go

```diff
@@ -15,6 +15,10 @@ var _ = context.Background
 {{- if .IsExported}}
 {{if and (is "Enum" .) .IsValueEnum}}
 {{$enumName := .Name -}}
+{{if .Comments -}}
```

Comment:
The output looks good, but let's add a `.Comments` field to all `Decl` implementations, and pull this out of each case here.

Reply:
This is types like `Config`, `Secret`, `Database`, etc.

Reply:
Will do in another PR